model-index.yml

```diff
@@ -0,0 +1,58 @@
+Import:
+  - configs/atss/metafile.yml
+  - configs/autoassign/metafile.yml
+  - configs/cascade_rcnn/metafile.yml
+  - configs/centernet/metafile.yml
+  - configs/centripetalnet/metafile.yml
+  - configs/cornernet/metafile.yml
+  - configs/dcn/metafile.yml
+  - configs/deformable_detr/metafile.yml
+  - configs/detectors/metafile.yml
+  - configs/detr/metafile.yml
+  - configs/double_heads/metafile.yml
+  - configs/dynamic_rcnn/metafile.yml
+  - configs/empirical_attention/metafile.yml
+  - configs/faster_rcnn/metafile.yml
+  - configs/fcos/metafile.yml
+  - configs/foveabox/metafile.yml
+  - configs/fp16/metafile.yml
+  - configs/fpg/metafile.yml
```

Comment:
What this file is used for? Are these paths in the fork or in the upstream repo? If it's local, do we have all of this?

Reply:
Good catch, it was only used in upstream, removed